### PR TITLE
Fyle 276nzt2 - Cards should be displayed in the dashboard and my expenses filter even if there aren't any expenses associated with it

### PR DESCRIPTION
**Pull request by saitejasFyle**

### CODE REVIEW

File: src/app/fyle/dashboard/stats/stats.component.ts [hunk 1]
```diff
@@ -134,9 +136,33 @@ export class StatsComponent implements OnInit {
...
       );
+
+      this.allCardTransactionsAndDetailsNonUnifyCCC$ = this.dashboardService.getNonUnifyCCCDetails().pipe(
+        map((res) => res),
+        shareReplay(1)
+      );
       this.cardTransactionsAndDetails$.subscribe((details) => {
         this.cardTransactionsAndDetails = this.getCardDetail(details.cardDetails);
-        this.isCCCStatsLoading = false;
+        this.allCardTransactionsAndDetailsNonUnifyCCC$.subscribe((cards) => {
```
Comment (Julias0) [inline src/app/fyle/dashboard/stats/stats.component.ts:146]: ok this is a subscribe inside a subscribe which is an antipattern. We should do some cleanup here

File: src/app/fyle/my-expenses/my-expenses.page.ts [hunk 2]
```diff
@@ -478,6 +490,21 @@ export class MyExpensesPage implements OnInit {
         cards.forEach((card) => {
           this.cardNumbers.push({ label: this.maskNumber.transform(card.cardNumber), value: card.cardNumber });
         });
+
+        this.allCardTransactionsAndDetailsNonUnifyCCC$.subscribe((details) => {
```
Comment (Julias0) [inline src/app/fyle/my-expenses/my-expenses.page.ts:494]: subscribe inside subscribe again

File: src/app/fyle/dashboard/stats/stats.component.ts (hunk 1 above)
Comment (saitejasFyle) [reply src/app/fyle/dashboard/stats/stats.component.ts]: Done

File: src/app/fyle/my-expenses/my-expenses.page.ts (hunk 2 above)
Comment (saitejasFyle) [reply src/app/fyle/my-expenses/my-expenses.page.ts]: Done

**Review by Julias0 — APPROVED** [review]

**Pull request merged**